definitions/SpellItemEnchantment.dbd

```diff
@@ -20,11 +20,10 @@ float EffectScalingPoints
 locstring HordeName_lang
 int<FileData::ID> IconFileDataID
 int TransmogCost
-int<PlayerCondition::ID> TransmogPlayerConditionID
+int<PlayerCondition::ID> TransmogPlayerConditionID // pre-9.0: condition to unlock as transmog illusion; post-9.0: condition to use as transmog illusion after unlocking
```

Comment:
Is it *really* the same column though, not one removed and one added?

Reply:
Not the same, the old one was moved to TransmogIllusion

Reply:
What about naming the old one `TransmogUnlockConditionID` and the new one `TransmogConditionID` or `TransmogUseConditionID`, to keep them explicit and separate?

Reply:
Changed this